src/Avalonia.Controls/DragDrop/DefaultDragDispatcher.cs

```diff
@@ -0,0 +1,90 @@
+﻿using Avalonia.Input;
+using Avalonia.Interactivity;
+using Avalonia.VisualTree;
+using System.Linq;
+
+namespace Avalonia.Controls.DragDrop
+{
+    class DefaultDragDispatcher : IDragDispatcher
+    {
+        public static readonly DefaultDragDispatcher Instance = new DefaultDragDispatcher();
+
+        private Interactive _lastTarget = null;
+        
+        private DefaultDragDispatcher()
+        {   
+        }
+
+        private Interactive GetTarget(IInputElement root, Point local)
+        {
+            var target = root.InputHitTest(local)?.GetSelfAndVisualAncestors()?.OfType<Interactive>()?.FirstOrDefault();
+            if (target != null && DragDrop.GetAcceptDrag(target))
```

Comment:
It should traverse the visual tree to check if there is an ancestor element, that accepts drag.

Reply:
This is implemented like the behavior of WPF. Since the AcceptDrag property is inherited, all child elements should accept drags if not explicitly disabled, in that case it should not give the drag data to the parent elements. (IMHO)